src/HonzaBotner.Discord.Services/Jobs/StandUpJobProvider.cs

```diff
@@ -79,34 +88,58 @@ await channel.GetMessagesBeforeAsync(messageList.Last().Id)
                 }
             }
 
-            foreach (DiscordMessage msg in messageList.Where(msg => msg.Timestamp.Date == yesterday))
+            foreach (var authorGrouped in messageList.Where(msg => msg.Timestamp.Date == yesterday)
+                         .GroupBy(msg => msg.Author.Id))
             {
-                foreach (Match match in Regex.Matches(msg.Content))
+                int total = 0;
+                int completed = 0;
+
+                foreach (var msg in authorGrouped)
                 {
-                    string state = match.Groups["State"].ToString();
-                    string priority = match.Groups["Priority"].ToString();
 
-                    if (OkList.Any(s => state.Contains(s)))
+                    foreach (Match match in TaskRegex.Matches(msg.Content))
```

Comment:
```suggestion
                    foreach (var match in TaskRegex.Matches(msg.Content))
```
to keep it consistent with previous loop.

Reply:
This actually turned everything red, so I kept it the way it is now.. For some reason once we substitute Mtch for var, it no longer knows that var contains some groups...